hwilib/psbt.py

```diff
@@ -185,6 +199,72 @@ def deserialize(self, f: Readable) -> None:
                     raise PSBTSerializationError("final scriptWitness key is more than one byte type")
                 witness_bytes = BufferedReader(BytesIO(deser_string(f))) # type: ignore
                 self.final_script_witness.deserialize(witness_bytes)
+            elif key_type == 0x13:
```

Comment:
42963840ebe71e8553681e5ce80b84de8ba939ec : Maybe define these keytypes as constants in one place?